### PR TITLE
Import and use standard models of inheritance defined in ClinVar constants

### LINKED ISSUE #2570: ClinVar- feed vack for correctionfrom ClinVar
Hi,
I've submitted variants that had a AD inheritance this the comment I've received bakc from clinVan about the foramt inf the excell file

"Mode of inheritance (col. M) I have corrected "Autosomal dominant" to the accepted value "Autosomal dominant inheritance" to allow processing. "

Could yo please adjust this? I have a a huge amount of AD variants to submitt (mostry of EP cases carry a AD variannt)

In the mean time I can correct it myself but it would be good to comply with the required format :)

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2570.

Description of ClinVar data fields is here: https://www.ncbi.nlm.nih.gov/projects/clinvar/ClinVarDataDictionary.pdf
In this document they mention that the models of inheritance to be used are in this document: https://ftp.ncbi.nlm.nih.gov/pub/GTR/standard_terms/Mode_of_inheritance.txt

**How to test**:
1. Pin a variant and check that the inheritance model field is displayed correctly in the ClinVar form.
2. Save submission and export it to file.
3. Check that inheritance model is correct on exported file or that is missing if no model was selected

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/clinvar.html [hunk 1]
```diff
@@ -175,56 +175,10 @@ <h5>
                       </td>
                       <td>
                         <select id="Mode of inheritance_{{pinned._id}}" name="inheritance_mode@{{pinned._id}}">
-                          {% if 'AR_hom' in pinned.genetic_models or 'AR_hom_dn' in pinned.genetic_models or 'AR_comp' in pinned.genetic_models or 'AR_comp_d' in pinned.genetic_models %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/clinvar.html:178]: 🤢

File: scout/server/blueprints/variant/controllers.py [hunk 2]
```diff
@@ -457,4 +458,5 @@ def clinvar_export(store, institute_id, case_name, variant_id):
         case=case_obj,
         variant=variant_obj,
         pinned_vars=pinned,
+        inherirance_models=INHERITANCE_MODELS,
```
Comment (dnil) [inline scout/server/blueprints/variant/controllers.py]:
```suggestion
        inheritance_models=INHERITANCE_MODELS,
```

File: scout/constants/clinvar.py [hunk 3]
```diff
@@ -102,3 +102,27 @@
     "practice_guideline",
     "reviewed_by_expert_panel",
 }
+
+# Models of inheritance are defined here:
+# https://ftp.ncbi.nlm.nih.gov/pub/GTR/standard_terms/Mode_of_inheritance.txt
+INHERITANCE_MODELS = [
```
Comment (dnil) [inline scout/constants/clinvar.py:108]: Could we preface that with like CLINVAR? One might be confused as GenMod uses a different set. (https://github.com/moonso/genmod/blob/master/docs/genetic-models.md)
Or name the clinvar one INHERITANCE_MODES and let the GenMod sphere keep "inheritance models". Then we have OMIM and HPO with same-same-but-diffrent. 🙄

File: scout/server/blueprints/variant/templates/variant/clinvar.html [hunk 4]
```diff
@@ -175,56 +175,10 @@ <h5>
...
-                          {% endif %}
-                          <option value="Autosomal unknown">Autosomal unknown</option>
-                          <option value="Codominant">Codominant</option>
-                          <option value="Genetic anticipation">Genetic anticipation</option>
-                          <option value="Oligogenic">Oligogenic</option>
-                          <option value="Somatic mutation">Somatic mutation</option>
-                          <option value="Sporadic">Sporadic</option>
-                          <option value="Other">Other</option>
+                          <option value="" selected>-</option>
+                          {% for model in inherirance_models %}
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/clinvar.html]:
```suggestion
                          {% for model in inheritance_models %}
```

File: scout/server/blueprints/variant/templates/variant/clinvar.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/clinvar.html:178]: 😂

**Review by dnil — APPROVED** [review]
🥇

File: scout/server/blueprints/variant/templates/variant/clinvar.html (hunk 4 above)
Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/clinvar.html]: Same here

File: scout/constants/clinvar.py (hunk 3 above)
Comment (northwestwitch) [reply scout/constants/clinvar.py:108]: Sure! CLINVAR_INHERITANCE_MODELS is more precise

**Comment by northwestwitch** [comment]
Renamed variant to CLINVAR_INHERITANCE_MODELS and still works, merging when it passes the tests!

**Pull request merged**